### DISCUSSION TITLE
Capture html output from Koj bot on 4xx or 5xx responses and attach to issue.

### DISCUSSION THREAD

**Discussion by rouilj** [upptime/upptime#880] [Ideas]
I have been a happy user of Upptime since Dec 2020. However every now and again I get a 503 error
from one of my services.

The service is running behind my main web server and is reverse proxied by that server. The logs for the proxy server report the 503 and show ~4k of data returned. Since I can't see the contents of the 503 response, I have no idea what caused the error.

Would it be possible to attach the HTML output to the issue opened for the 503 so I can get some idea as to what is going wrong?

Thanks.

-- rouilj

**Comment by yarikoptic** [comment]
I would have loved to see that too for a regular sensing of a website -- we get sporadic 503s and (smile not) having difficulties to access server-side logs ATM. So seeing at least what server returned along with the http return code would have been informative.  Any way to achieve that with ease? ;-)
